### PR TITLE
Ems

### PR DESCRIPTION
**Pull request by EMSwank**
Fix houndcl errors

### CODE REVIEW

File: lib/enigma.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+require 'pry'
+require './lib/dictionary'
+
+class Enigma
+  include Dictionary
+  def random_key(length, ceiling)
+    length.times.map { Random.rand(ceiling) }
+  end
+
+  # problematic for edge cases? can we hard code in random_key and still have appropriate tests
```
[the remark this answers is not in the document]
Comment (agpiermarini) [reply lib/enigma.rb]: Comment will be deleted in future iterations
